packages/components/src/InputFile/InputFile.css

```
@@ -22,3 +22,13 @@
   border-color: var(--color-focus);
   background-color: var(--color-surface--hover);
 }
+
+.dropZone.error {
+  border: var(--color-critical) dashed var(--border-thick);
```

Only need to specificy border-color here:

```suggestion
  border-color: var(--color-critical);
```